Frameworks/UIKit/UIPopoverPresentationController.mm

```diff
@@ -316,7 +325,7 @@ - (void)setPopoverBackgroundViewClass:(Class)popoverBackgroundViewClass {
 }
 
 - (void)dealloc {
-    [_willDismissCompletion release];
+    [_onDismissCleanupHandler release];
```

Comment:
>[_onDismissCleanupHandler release]; [](start = 4, length = 35)

Do we need to set to nil as well? It's possible this has already happened if _handleDismissCleanup was called.